Tests failed:

```
running go unit test ./... + coverage in /home/circleci/project/exporter/awsprometheusremotewriteexporter
--- FAIL: TestLoadConfig (0.01s)
    config_test.go:104: 
        	Error Trace:	config_test.go:104
        	Error:      	Not equal: 
        	            	expected: &awsprometheusremotewriteexporter.Config{Config:prometheusremotewriteexporter.Config{ExporterSettings:(*config.ExporterSettings)(0xc00066e260), TimeoutSettings:exporterhelper.TimeoutSettings{Timeout:5000000000}, QueueSettings:exporterhelper.QueueSettings{Enabled:false, NumConsumers:0, QueueSize:0}, RetrySettings:exporterhelper.RetrySettings{Enabled:true, InitialInterval:10000000000, MaxInterval:60000000000, MaxElapsedTime:600000000000}, Namespace:"test-space", ExternalLabels:map[string]string{"key1":"value1", "key2":"value2"}, HTTPClientSettings:confighttp.HTTPClientSettings{Endpoint:"http://localhost:9009", TLSSetting:configtls.TLSClientSetting{TLSSetting:configtls.TLSSetting{CAFile:"/var/lib/mycert.pem", CertFile:"", KeyFile:""}, Insecure:false, InsecureSkipVerify:false, ServerName:""}, ReadBufferSize:0, WriteBufferSize:524288, Timeout:5000000000, Headers:map[string]string{"prometheus-remote-write-version":"0.1.0", "x-scope-orgid":"234"}, CustomRoundTripper:(func(http.RoundTripper) (http.RoundTripper, error))(nil)}}, AuthConfig:awsprometheusremotewriteexporter.AuthConfig{Region:"us-west-2", Service:"service-name", RoleArn:"arn:aws:iam::123456789012:role/IAMRole"}}
        	            	actual  : &awsprometheusremotewriteexporter.Config{Config:prometheusremotewriteexporter.Config{ExporterSettings:(*config.ExporterSettings)(0xc00066ec80), TimeoutSettings:exporterhelper.TimeoutSettings{Timeout:5000000000}, QueueSettings:exporterhelper.QueueSettings{Enabled:true, NumConsumers:2, QueueSize:10}, RetrySettings:exporterhelper.RetrySettings{Enabled:true, InitialInterval:10000000000, MaxInterval:60000000000, MaxElapsedTime:600000000000}, Namespace:"test-space", ExternalLabels:map[string]string{"key1":"value1", "key2":"value2"}, HTTPClientSettings:confighttp.HTTPClientSettings{Endpoint:"http://localhost:9009", TLSSetting:configtls.TLSClientSetting{TLSSetting:configtls.TLSSetting{CAFile:"/var/lib/mycert.pem", CertFile:"", KeyFile:""}, Insecure:false, InsecureSkipVerify:false, ServerName:""}, ReadBufferSize:0, WriteBufferSize:524288, Timeout:5000000000, Headers:map[string]string{"prometheus-remote-write-version":"0.1.0", "x-scope-orgid":"234"}, CustomRoundTripper:(func(http.RoundTripper) (http.RoundTripper, error))(nil)}}, AuthConfig:awsprometheusremotewriteexporter.AuthConfig{Region:"us-west-2", Service:"service-name", RoleArn:"arn:aws:iam::123456789012:role/IAMRole"}}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -10,5 +10,5 @@
        	            	   QueueSettings: (exporterhelper.QueueSettings) {
        	            	-   Enabled: (bool) false,
        	            	-   NumConsumers: (int) 0,
        	            	-   QueueSize: (int) 0
        	            	+   Enabled: (bool) true,
        	            	+   NumConsumers: (int) 2,
        	            	+   QueueSize: (int) 10
        	            	   },
        	Test:       	TestLoadConfig
FAIL
```

--- a/exporter/awsprometheusremotewriteexporter/config_test.go
+++ b/exporter/awsprometheusremotewriteexporter/config_test.go
@@ -60,11 +60,6 @@ func TestLoadConfig(t *testing.T) {
 				TypeVal: "awsprometheusremotewrite",
 			},
 			TimeoutSettings: exporterhelper.DefaultTimeoutSettings(),
-			QueueSettings: exporterhelper.QueueSettings{
-				Enabled:      true,
-				NumConsumers: 2,
-				QueueSize:    10,
-			},
 			RetrySettings: exporterhelper.RetrySettings{
 				Enabled:         true,
 				InitialInterval: 10 * time.Second,
@@ -81,12 +76,9 @@ func TestLoadConfig(t *testing.T) {
 					},
 					Insecure: false,
 				},
-				ReadBufferSize: 0,
-
+				ReadBufferSize:  0,
 				WriteBufferSize: 512 * 1024,
-
-				Timeout: 5 * time.Second,
-
+				Timeout:         5 * time.Second,
 				Headers: map[string]string{
 					"prometheus-remote-write-version": "0.1.0",
 					"x-scope-orgid":                   "234"},

